causing/estimate.py

```diff
@@ -663,12 +678,19 @@ def estimate_snn(model_dat, do_print=True):
     optimizer = torch.optim.Rprop(params)
 
     if do_print:
-        print(
-            "\nEstimation of direct effects using a structural neural network \n"
-            "with regularization parameter alpha = {:10f}:".format(model_dat["alpha"])
-        )
+        print("\nEstimation of direct effects using a structural neural network.")
```

Comment:
Why do not print alpha here?

Reply:
Because `alpha` is only part of the estimation input and not part of `model_dat`, anymore. If we want to print it, we have to either do it as part of the estimation, or return it back from the estimation. I went the simple route of skipping it for now instead of thinking about the best way to do it.